src/screens/Home/ActionButtons.js

```diff
@@ -279,11 +301,20 @@ class ActionButtons extends React.Component<Props, State> {
             />
           </ActionButtonsWrapper>
         </Sizer>
+        {visibleActionModal === RECEIVE &&
```

Comment:
Why double visibility setting is needed here? 
Wouldn't `isVisible={visibleActionModal === RECEIVE}` be enough?